Cargo.toml

```diff
@@ -19,21 +19,21 @@ std = ["alloc"]
 default = ["std"]
 
 [dependencies]
-futures-core = { version = "0.3.0", default-features = false }
-lock_api = "0.3.2"
-parking_lot = { version = "0.10", optional = true }
+futures-core = { version = "0.3.5", default-features = false }
+lock_api = "0.4.1"
+parking_lot = { version = "0.11.0", optional = true }
 
 [dev-dependencies]
-futures = { version = "0.3.0", default-features = true, features=["async-await"] }
-futures-test = { version = "0.3.0", default-features = true }
-pin-utils = "0.1.0-alpha.4"
-criterion = "0.3.0"
-crossbeam = "0.7" # For channel benchmarks
+futures = { version = "0.3.5", default-features = true, features = ["async-await"] }
+futures-test = { version = "0.3.5", default-features = true }
+pin-utils = "0.1.0"
+criterion = "0.3.3"
```

Comment:
This, crossbeam, and rand should be pushed to the newest version with the same minor number anyway. Don't see a lot of benefit in tracking fix versions.

No concern regarding more explicit updates of async-std and tokio however, since there seem to have been quite a bit of changes.